CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.10.0
+
+* Drop `--allow-missing-relations-in-first-blocks` flag
+* Introduce `--non-strict-mode` which does the same as `--allow-missing-relations-in-first-blocks` flag did but endlessly
+* Add `--stop-after-number-of-blocks <amount>` flag to plan Indexer for Explorer to stop once it indexed the provided `<amount>` of blocks. May be useful for debug or maintenance purposes.
```

Comment:
```suggestion
* Add `--stop-after-number-of-blocks <count>` flag to plan Indexer for Explorer to stop once it indexed the provided `<count>` of blocks. May be useful for debug or maintenance purposes.
```

Use `amount` for non-individually countable nouns (mass, money, tokens, etc) and `count`/`number` for individually countable (blocks, transactions, people)